tests/Tests.FeatureManagement/Tests.FeatureManagement.csproj

```diff
@@ -1,32 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
>netcoreapp2.1; [](http://example.com/codeflow?start=22&length=14)

Since we removed netcoreapp2.1 from target frameworks, are we deprecating netstandard2.0 support with this change?

Reply:
That's a good question. netcoreapp2.1 is no longer supported which is why I thought to remove it since I was updating tests. Your comment made me realize that by removing the netcoreapp2.1 target framework we are no longer testing the netstandard2.0 binary.

Support for netstandard 2.0 going forward is up for discussion, but for now in this version of Microsoft.FeatureManagement (2.x) we still have it so I still want netcoreapp2.1 to be testing that binary. I added back the netcoreapp2.1 target framework to avoid convoluting this PR.